src/azure-account.api.d.ts

```diff
@@ -0,0 +1,33 @@
+import { Event } from 'vscode';
```

Comment:
hmmm, there's not an existing `typings` folder for things like this?  Would be great to hand this back to owners of azure account so they can embed correct typings in their module.

Reply:
I agree it's annoying. However, we mitigated this issue by only having 'azure-account.api.d.ts' in our UI package.

Please remove this file and use IAzureNode.userId if you really need the azure userId
https://github.com/Microsoft/vscode-azuretools/pull/49

Reply:
I've published the 0.4.1 version of the ui package. You should be able to get userId now.

Reply:
Sounds good. I will rework the PR. Thanks for taking care of that.